database/migrations/2020_12_03_151728_create_applicants_skills_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateApplicantsSkillsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('applicants_skills', function (Blueprint $table) {
```

Comment:
The default pivot table name should use the singular model names (applicant_skill). 
https://laravel.com/docs/8.x/eloquent-relationships#many-to-many-table-structure